Installation instructions in the readme

I followed the installation instructions listed in the README and they worked. However, the first command fails with

```
# gopkg.in/bblfsh/client-go.v2/tools
In file included from ../../bblfsh/client-go.v2/tools/filter.go:14:0:
./bindings.h:8:18: fatal error: uast.h: No such file or directory
compilation terminated.
```

I understand that this is intended because `make deps` fixes it. Should we put a notice there in the README that the first command will fail and it is normal?

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ services:
   - docker
 
 before_install:
-  - docker run --privileged -d -p 9432:9432 --name bblfsh bblfsh/server
+  - docker run --privileged -d -p 9432:9432 --name bblfsh bblfsh/bblfshd
   - make dependencies
 
 script:

--- a/README.md
+++ b/README.md
@@ -33,7 +33,7 @@ This small example illustrates how to retrieve the [UAST](https://doc.bblf.sh/ua
 If you don't have a bblfsh server running you can execute it using the following command:
 
 ```sh
-docker run --privileged --rm -it -p 9432:9432 --name bblfsh bblfsh/server
+docker run --privileged --rm -d -p 9432:9432 --name bblfsh bblfsh/bblfshd
 ```
 
 Please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server.

--- a/README.md
+++ b/README.md
@@ -9,11 +9,22 @@ and to analyse UASTs with the functionality provided by [libuast](https://github
 
 The recommended way to install *client-go* is:
 
-```
+```sh
 go get -u gopkg.in/bblfsh/client-go.v2/...
 cd $GOPATH/src/gopkg.in/bblfsh/client-go.v2
 make dependencies
 ```
+
+The first command will fail when trying to compile the project because we stil
+doesn't have some dependencies with:
+
+```sh
+In file included from tools/filter.go:14:0:
+./bindings.h:8:10: fatal error: uast.h: No such file or directory
+ #include "uast.h"
+```
+
+But the `make dependencies` command will compile and install everything.
 
 ## Example
 

--- a/README.md
+++ b/README.md
@@ -10,21 +10,10 @@ and to analyse UASTs with the functionality provided by [libuast](https://github
 The recommended way to install *client-go* is:
 
 ```sh
-go get -u gopkg.in/bblfsh/client-go.v2/...
+go get -d -u gopkg.in/bblfsh/client-go.v2/...
 cd $GOPATH/src/gopkg.in/bblfsh/client-go.v2
 make dependencies
 ```
-
-The first command will fail when trying to compile the project because we stil
-doesn't have some dependencies with:
-
-```sh
-In file included from tools/filter.go:14:0:
-./bindings.h:8:10: fatal error: uast.h: No such file or directory
- #include "uast.h"
-```
-
-But the `make dependencies` command will compile and install everything.
 
 ## Example
 

